src/celengine/namedb.cpp

```diff
@@ -82,6 +82,13 @@ std::vector<Name> NameDatabase::getCompletion(const std::string& name, bool gree
             completion.push_back(iter->first);
         }
     }
+    for (NameMap::const_iterator iter = m_localizedIndex.begin(); iter != m_localizedIndex.end(); ++iter)
```

Comment:
```
for(const auto& local : m_localizedIndex)
```
?

Reply:
Indeed. Just copied existing code and forgot about optimisations.